content/pop-recovery.md

```diff
@@ -156,7 +156,7 @@ Alternatively you can also upgrade the recovery partition from the command-line
 pop-upgrade recovery upgrade from-release
 ```
 
-You should see an output similar to below saying the recovery parition had been updated:
+You should see an output similar to below saying the recovery partition had been updated:
```

Comment:
*You should see an output similar to what is pictured below...

OR 

to what is included below

Etc.